assert/assertions.go

```diff
@@ -705,7 +705,7 @@ func Empty(t TestingT, object interface{}, msgAndArgs ...interface{}) bool {
 		if h, ok := t.(tHelper); ok {
 			h.Helper()
 		}
-		Fail(t, fmt.Sprintf("Should be empty, but was %v", object), msgAndArgs...)
+		Fail(t, fmt.Sprintf("Should be empty, but isn't %v", object), msgAndArgs...)
```

Comment:
The message shows the unexpected value. `is` is ok.
```suggestion
		Fail(t, fmt.Sprintf("Should be empty, but is %v", object), msgAndArgs...)
```